Unable to build Moq: "The `Microsoft.Build.Tasks.Git.LocateRepository` task failed unexpectedly."

Hello,

Trying to build this project in VS 2019 (16.7.4), results in the following errors:

![image](https://user-images.githubusercontent.com/2716316/94351890-44c00b00-0067-11eb-9caa-ec283a1c25a9.png)

I have the following SDKs installed:

![image](https://user-images.githubusercontent.com/2716316/94351961-d465b980-0067-11eb-96cb-a8b1ab0d1ff9.png)

Am I missing any prerequisite?

Trying `dotnet build` results in the following output:

```
❯ dotnet build
Microsoft (R) Build Engine version 16.7.0-preview-20360-03+188921e2f for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  Restored D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests.FSharpTypes\Moq.Tests.FSharpTypes.fsproj (in 125 ms).
  Restored D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests\Moq.Tests.csproj (in 349 ms).
  2 of 4 projects are up-to-date for restore.
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(ObjectHandleOnStack assemblyName, ObjectHandleOnStack requestingAssembly, StackCrawlMarkHandle stackMark, Boolean throwOnFileNotFound, ObjectHandleOnStack assemblyLoadContext, ObjectHandleOnStack retAssembly) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(AssemblyName assemblyName, RuntimeAssembly requestingAssembly, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(ObjectHandleOnStack assemblyName, ObjectHandleOnStack requestingAssembly, StackCrawlMarkHandle stackMark, Boolean throwOnFileNotFound, ObjectHandleOnStack assemblyLoadContext, ObjectHandleOnStack retAssembly) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(AssemblyName assemblyName, RuntimeAssembly requestingAssembly, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
  Moq.Tests.FSharpTypes -> D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests.FSharpTypes\bin\Release\net472\Moq.Tests.FSharpTypes.dll
  Moq.Tests.FSharpTypes -> D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests.FSharpTypes\bin\Release\netcoreapp3.1\Moq.Tests.FSharpTypes.dll

Build FAILED.

C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(ObjectHandleOnStack assemblyName, ObjectHandleOnStack requestingAssembly, StackCrawlMarkHandle stackMark, Boolean throwOnFileNotFound, ObjectHandleOnStack assemblyLoadContext, ObjectHandleOnStack retAssembly) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(AssemblyName assemblyName, RuntimeAssembly requestingAssembly, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(ObjectHandleOnStack assemblyName, ObjectHandleOnStack requestingAssembly, StackCrawlMarkHandle stackMark, Boolean throwOnFileNotFound, ObjectHandleOnStack assemblyLoadContext, ObjectHandleOnStack retAssembly) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoad(AssemblyName assemblyName, RuntimeAssembly requestingAssembly, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
    0 Warning(s)
    2 Error(s)

Time Elapsed 00:00:05.82
```

After spending a nice half an hour uninstalling and installing some versions of .NET, I still can't get it to work.
I'm getting the same errors.

Here's my current output, what am I missing?

```
Microsoft (R) Build Engine version 16.7.0+7fb82e5b2 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  Restored D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests.FSharpTypes\Moq.Tests.FSharpTypes.fsproj (in 219 ms).
  Restored D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests\Moq.Tests.csproj (in 379 ms).
  2 of 4 projects are up-to-date for restore.
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly assemblyContext, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.Load(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly assemblyContext, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.Load(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
  Moq.Tests.FSharpTypes -> D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests.FSharpTypes\bin\Release\net472\Moq.Tests.FSharpTypes.dll
  Moq.Tests.FSharpTypes -> D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\tests\Moq.Tests.FSharpTypes\bin\Release\netcoreapp3.1\Moq.Tests.FSharpTypes.dll

Build FAILED.

C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly assemblyContext, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.Load(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: File name: 'Microsoft.Build.Tasks.Git, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly assemblyContext, StackCrawlMark& stackMark, Boolean throwOnFileNotFound, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.Load(AssemblyName assemblyRef, StackCrawlMark& stackMark, AssemblyLoadContext assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.GitLoaderContext.Load(AssemblyName assemblyName) in /_/src/Microsoft.Build.Tasks.Git/GitLoaderContext.cs:line 24 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.ResolveUsingLoad(AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Runtime.Loader.AssemblyLoadContext.Resolve(IntPtr gchManagedAssemblyLoadContext, AssemblyName assemblyName) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at System.Reflection.Assembly.GetType(String name, Boolean throwOnError) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.TaskImplementation..cctor() in /_/src/Microsoft.Build.Tasks.Git/TaskImplementation.cs:line 35 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:  [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    --- End of inner exception stack trace --- [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.Tasks.Git.LocateRepository.Execute() in /_/src/Microsoft.Build.Tasks.Git/LocateRepository.cs:line 18 [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
C:\Users\Shimmy\.nuget\packages\microsoft.build.tasks.git\1.0.0-beta2-18618-05\build\Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [D:\Users\Shimmy\Source\Repos\weitzhandler\moq4\src\Moq\Moq.csproj]
    0 Warning(s)
    2 Error(s)

Time Elapsed 00:00:05.22
```

--- a/build/SourceLink.props
+++ b/build/SourceLink.props
@@ -6,7 +6,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="All"/>
+		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
 	</ItemGroup>
 
 </Project>

